tests/applications.js

```diff
@@ -380,7 +388,7 @@ test('Send - bad urls', t => {
     t.true(Array.isArray(app.responses.sunset), 'includes live responses, which is an array');
 
     const sorted = _.sortBy(app.responses.sunset, 'timestamp');
-    t.is(sorted[sorted.length - 1].response, 500, 'includes endpoint response');
+    t.is(sorted[sorted.length - 1].response, 400, 'includes endpoint response');
```

Comment:
This is consistently failing for me with a `400` on local, but `500` on Travis. I don't understand